src/pretix/control/forms/global_settings.py

```diff
@@ -13,215 +13,343 @@
 
 
 class GlobalSettingsForm(SettingsForm):
-    auto_fields = [
-        'region',
-        'mail_from'
-    ]
+    auto_fields = ["region", "mail_from"]
+
+    # Define which fields belong to which tabs
+    tab_structure = {
+        "basics": [
+            "footer_text",
+            "footer_link",
+            "banner_message",
+            "banner_message_detail",
+            "ticket_fee_percentage",
+        ],
+        "localization": [
+            "region",
+        ],
+        "email": [
+            "mail_from",
+            "email_vendor",
+            "send_grid_api_key",
+            "smtp_host",
+            "smtp_port",
+            "smtp_username",
+            "smtp_password",
+            "smtp_use_tls",
+            "smtp_use_ssl",
+        ],
+        "payment_gateways": [
+            "payment_stripe_connect_secret_key",
+            "payment_stripe_connect_publishable_key",
+            "payment_stripe_connect_test_secret_key",
+            "payment_stripe_connect_test_publishable_key",
+            "stripe_webhook_secret_key",
+        ],
+        "maps": [
+            "opencagedata_apikey",
+            "mapquest_apikey",
+            "leaflet_tiles",
+            "leaflet_tiles_attribution",
+        ],
+    }
 
     def _setting_default(self):
         """
-            Load default email setting form .cfg file if not set
+        Load default email setting form .cfg file if not set
         """
         global_settings = self.obj.settings
-        if global_settings.get('smtp_port') is None or global_settings.get('smtp_port') == "":
-            self.obj.settings.set('smtp_port', settings.EMAIL_PORT)
-        if global_settings.get('smtp_host') is None or global_settings.get('smtp_host') == "":
-            self.obj.settings.set('smtp_host', settings.EMAIL_HOST)
-        if global_settings.get('smtp_username') is None or global_settings.get('smtp_username') == "":
-            self.obj.settings.set('smtp_username', settings.EMAIL_HOST_USER)
-        if global_settings.get('smtp_password') is None or global_settings.get('smtp_password') == "":
-            self.obj.settings.set('smtp_password', settings.EMAIL_HOST_PASSWORD)
-        if global_settings.get('smtp_use_tls') is None or global_settings.get('smtp_use_tls') == "":
-            self.obj.settings.set('smtp_use_tls', settings.EMAIL_USE_TLS)
-        if global_settings.get('smtp_use_ssl') is None or global_settings.get('smtp_use_ssl') == "":
-            self.obj.settings.set('smtp_use_ssl', settings.EMAIL_USE_SSL)
-        if global_settings.get('email_vendor') is None or global_settings.get('email_vendor') == "":
-            self.obj.settings.set('email_vendor', 'smtp')
+        if (
+            global_settings.get("smtp_port") is None
+            or global_settings.get("smtp_port") == ""
+        ):
```

Comment:
You seem to have a different line length setting, which introduces a lot of unrelated changes.
Please change this and make sure that no code changes are mixed with reformatting issues.